modules/argocd-console-sample.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/using-argocd.adoc
+
+[id="argocd-console-sample_{context}"]
+= Web console
+
+You can use ArgoCD to deploy web console modifications. The `./console` directory contains a simple configuration for the OpenShift console which simply changes the logout behavior to redirect to Google.
+
+.ArgoCD web console configuration sample (`console.yaml`)
+----
+---
+apiVersion: config.openshift.io/v1
+kind: Console
+metadata:
+  name: cluster
+  annotations:
+    argocd.argoproj.io/sync-options: Prune=false
```

Comment:
This won't be needed as pruning it's not being enabled when creating the Argo CD App.